packages/api-client/src/main/team/invitation/TeamInvitationAPI.ts

```diff
@@ -42,9 +43,41 @@ class TeamInvitationAPI {
     return this.client.sendJSON(config).then((response: AxiosResponse) => response.data);
   }
 
-  public getInvitations(teamId: string): Promise<TeamInvitationChunk> {
+  public async getAllInvitations(teamId: string): Promise<TeamInvitation[]> {
```

Comment:
Instead of having a `getAllInvitations` and a `getInvitations` you can simply overload functions:

```ts
public async getAllInvitations(teamId: string): Promise<TeamInvitation[]>
public async getAllInvitations(teamId: string, startEmail?: string, limit = TeamInvitationAPI.MAX_CHUNK_SIZE): Promise<TeamInvitation[]> { ... }
```

A PR of @ffflorian shows how to do this:
- https://github.com/wireapp/wire-web-packages/pull/734/files#diff-7659fc1e7b17e608837c4c101bd880ea